### ISSUE TITLE
Unrecognized cop or department RSpecRails/InferredSpecType for V3.0.1

### ISSUE THREAD

**Issue by Franx0** [rubocop/rubocop-rspec#1928]
**Versions**  

rubocop (1.64.1)
rubocop-rspec (3.0.1)

**Command** 
```
bundle exec rubocop --format progress --config .rubocop_with_todo.yml
```

**Error output**
```
Error: unrecognized cop or department RSpecRails/InferredSpecType found in .rubocop_with_todo.yml
Did you mean 'RSpec/RepeatedDescription'?
```

Trying to update from version `2.30.0` to `3.1.0` unrecognized cop error is raised during build process.
Rubocop Rspec documentation link [here](https://docs.rubocop.org/rubocop-rspec_rails/cops_rspecrails.html#rspecrailsinferredspectype)

**Comment by ydah** [comment]
Can you tell me the result of the following command?

`bundle exec rubocop -V`.

**Comment by Franx0** [comment]
> bundle exec rubocop -V
```
1.64.1 (using Parser 3.3.3.0, rubocop-ast 1.31.3, running on ruby 3.1.4) [arm64-darwin20]
  - rubocop-performance 1.21.0
  - rubocop-rspec 3.0.1
  ```

**Comment by Franx0** [comment]
> Have you included rubocop-rspec_rails to your Gemfile and required it from your .rubocop.yml?

No, I didn't I've just installed:
```
  gem "rubocop"
  gem "rubocop-performance"
  gem "rubocop-rspec"
``` 

Do I need it @pirj ?

**Comment by pirj** [comment]
Yes, we have an [upgrade doc](https://docs.rubocop.org/rubocop-rspec/upgrade_to_version_3.html).
Any improvements to the discoverability of this document are welcome.

**Comment by Franx0** [comment]
Thank you so much @pirj @ydah for your support. It's working as expected now.
Just to confirm. In changelog the reference to this gem seems to be needed in case we use the gem but as we've just seen it is required.
Thank you one more time 👍🏻

**Comment by jeppester** [comment]
I wasted some time on this for a dependabot PR that upgraded to v3.
What confused me was that the output from running rubocop was leading to a dead end:
1. Error: The `RSpec/Rails/InferredSpecType` cop has been moved to `RSpecRails/InferredSpecType`.
2. Error: unrecognized cop or department RSpecRails/InferredSpecType found in .rubocop.yml
3. ?

I've created a PR which will I hope will lead those updating the gem in a better direction.
